docs/subtitles-and-captions.html

```diff
@@ -388,13 +396,15 @@ <h3 class="mt-4">Example Code:</h3>
         const textTracks = {
           options: {
             // theme: "", // one of 'default', 'videojs-default', 'yellow-outlined', 'player-colors' & '3d'
-            // position: "", // i.e. 'top-left', 'center' etc
-            // offset: { // Object of top/right/bottom/left
-            //   top: "25%",
-            //   left: "12%"
-            // },
             // fontFace: "", // any Google font
             // fontSize: "", // any CSS value
+            // gravity: "", // i.e. 'top-left', 'center' etc
+            // box: { // Object of x/y/width/height
+            //   x: "0%",
+            //   y: "0%",
+            //   width: "100%",
+            //   height: "100%"
```

Comment:
what is the usage of declaring width, height? i mean what use case it solves for you, does it provide an over-flowing responsive container?

Reply:
It allows you to shift the box using x/y while making it smaller to make sure it doesn't overflow